Development/src/pages/queryapis/QueryAPIsList.js

```diff
@@ -0,0 +1,32 @@
+import React, { Fragment } from 'react';
+import { Datagrid, List, ShowButton, TextField } from 'react-admin';
+import ConnectButton from '../../components/ConnectButton';
+
+const QueryAPIsTitle = ({ record }) => {
+    return <span>Query APIs{record ? `: ${record.name}` : ''}</span>;
```

Comment:
Just noting that this is different in style to the equivalent code in the NMOS Resources, so if we're to add `ResourceTitle` as a component, let's work out why (no good reason, I'm guessing!)

Reply:
As this is the outlier however, maybe it would be worth making a separate title function the exception rather than the norm? It's not hurting anyone how it is, just a personal distaste for unnecessary repetition.